.gitignore

```diff
@@ -208,6 +208,7 @@ tags
 
 /test/library/packages/HDF5/ex_lite1.h5
 /test/library/packages/HDF5/ex_lite3.h5
+/test/library/packages/ZMQ/getLastEndpointHelper
```

Comment:
I suspect we all do it differently but I usually put such things in a directory-local .gitignore and CLEANFILES files. E.g. in test/library/packages/ZMQ/.gitignore

Reply:
I prefer the top level one myself.  I tend to only notice stuff hasn't been properly .gitignored long after the fact, so find it more trouble than it is worth to track down the original directory with the .gitignore.  If everything is in the top level, I can just update one place and it is less likely there is an old .gitignore reference that will never get cleaned up.